Ошибка: HTTPConnectionPool(host='a2z.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002C3936161D0>: Failed to resolve 'a2z.com' ([Errno 11001] getaddrinfo failed)"))